.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,23 @@
+name: Node.js CI
+
+on: [push, pull_request]
```

Comment:
could do on push to master, and all PRs, so we don't get what happened here - both for PR and for push